gpu_bdb/benchmark_runner.py

```diff
@@ -47,6 +57,11 @@ def load_query(qnum, fn):
 
     client, c = attach_to_cluster(config, create_sql_context=include_sql)
     # Preload required libraries for queries on all workers
+    import dask_sql
+    dask_sql_modules = [m for m in sys.modules if m.startswith('dask_sql')]
+    client.run(prevent_sql_import, dask_sql_modules)
+    print("is_jvm_started", client.run(is_jvm_started))
```

Comment:
Can this section be moved under the dask_sql runs under `if include_sql`

Reply:
I was worried that we might import `dask-sql` with `client.run(import_query_libs)` , cause those have `dask-sql` dependency's . 

I would like to keep this as is unless we have strong reasons to move it.

Reply:
In that case could we instead move the import dask_sql into a try catch block. The idea is we don't want users who built and environment from the non dask sql yaml file to run into issues if they aren't looking to run sql queries

Reply:
Ahh, gotcha. That we can do.  Let me update with that in mind.

Reply:
Resolved by https://github.com/VibhuJawa/gpu-bdb/commit/d5ed12f40734bb7f5968051e5e661ead559062c0